feconf.py

```diff
@@ -505,6 +506,11 @@ def get_empty_ratings():
 
 # Unfinished features.
 SHOW_TRAINABLE_UNRESOLVED_ANSWERS = False
+# Number of unresolved answers to be displayed in the dashboard for each
+# exploration.
+TOP_UNRESOLVED_ANSWERS_COUNT_DASHBOARD = 3
```

Comment:
All are fetched by only a single `get_multi` query, and then limited to 3 before sending in the payload to frontend: https://github.com/oppia/oppia/blob/6bae5592f79810e04474a9fe501fb0b9fc707db4/core/domain/stats_services.py#L89-L90